ci/run_envoy_docker.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
If we are cloning this file, we should clone it from the original in the Envoy repo:

https://github.com/envoyproxy/envoy/blob/main/ci/run_envoy_docker.sh

We should also document the maintenance procedure in a `MAINTAINERS.md` file indicating that this file needs to be cloned regularly, since Envoy changes it from time to time.

Example from Nighthawk:
https://github.com/envoyproxy/nighthawk/blob/main/MAINTAINERS.md

Reply:
Looks like the PR clones a few files from a couple of locations. When updating the MAINTAINERS.md, please list all the files we cloned and their original locations, so we know how to update this when necessary.